{{ cookiecutter.repo_name }}/pyproject.toml

```diff
@@ -33,24 +33,33 @@ markers = [
 ]
 
 [tool.pydocstyle]
+ignore = """
+"""
```

Comment:
you probably want to remove the `pydocstyle` section here, then add `ruff` & remove `flake8` & `pydocstyle` to/from the `dev-dependencies` above